styles/generic/_fonts.scss

```diff
@@ -0,0 +1,2 @@
+// any necessary @font-face declarations,
+// and those are imported by the SCSS entrypoint
```

Comment:
You can delete this file. With the font licensing issues we discussed previously, we won't be self-hosting any fonts. See https://sparkbox.atlassian.net/browse/FSA22V1-205 for details.

Reply:
Awesome, got it done!